Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -920,6 +920,70 @@ String ReplaceAllTextCommand::action_text() const
     return m_action_text;
 }
 
+IndentSelection::IndentSelection(TextDocument& document, TextRange const& range)
+    : TextDocumentUndoCommand(document)
+    , m_range(range)
+{
+}
+
+void IndentSelection::redo()
+{
+    if (m_tab_width == 0)
+        m_tab_width = m_client->soft_tab_width();
+
+    StringBuilder builder;
+    for (size_t i = 0; i < m_tab_width; i++)
+        builder.append(' ');
+
+    auto const tab = builder.build();
```

Comment:
You can simplify this:
```suggestion
    auto const tab = String::repeated(' ', m_tab_width);
```

Same for unindenting.

Reply:
Oo, nice. Changed it, thanks.